UsbAccess/UsbAccess.json

```diff
@@ -26,17 +26,12 @@
     },
     "methods": {
         "clearLink": {
-            "summary": "Clears or removes the symbolic link created by the `createLink` method",
+            "summary": "Clears or removes the symbolic link created by the `createLink` method. On failure, response returns error message and status of cleared link.\n \n### Events \n\n No Events.",
             "result": {
                 "type": "object",
                 "properties": {
                     "success":{
                         "$ref": "#/definitions/success"
-                    },
-                    "error":{
```

Comment:
why you are removing the "error" property from all responses?

Reply:
Updated the changes as per your  review comments.

Reply:
looks good